gce/scripts/startup-script.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START getting_started_gce_startup_script]
+set -e
+export HOME=/root
+
+# Install PHP and dependencies from apt
+apt-get update
+apt-get install -y git nginx php7.2 php7.2-fpm php7.2-mysql php7.2-dev \
```

Comment:
Should this be on PHP 7.3?

Reply:
PHP 7.3 is not available as an `apt-get` package on this version of ubuntu